app/controllers/api/v1/subscription_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
   end
 
   def update
-    render json: SubscriptionSerializer.new(Subscription.update(params[:id], subscription_params))
+    render json: SubscriptionSerializer.new(Subscription.update(params[:id], subscription_params), include: [:shipments])
```

Comment:
This is probably not very efficient, will result in N+1 queries as the app scales. My idea is to use a model method that limits the results to the most recent 5 shipments